src/main/java/factionsplusplus/services/CommandService.java

```diff
@@ -208,7 +208,7 @@ public boolean processCommand(Command parentCommand, Command command, CommandSen
             }
         }
 
-        if (command.shouldRequireNoFactionMembership()) {
+        if (! command.shouldRequireNoFactionMembership()) {
```

Comment:
I don't think this is right. If a command should require a player to not be in a faction (which is what this parameter tells us), we'd want to check that the player isn't in a faction. The code enclosed in the conditional does that

Reply:
Maybe `PlayerService`? Seems a bit more fitting

Reply:
wtf that was my cat, it somehow pasted this